code/modules/power/singularity/singularity.dm

```diff
@@ -94,6 +94,29 @@
 /obj/machinery/singularity/Crossed(atom/movable/A)
 	consume(A)
 
+/obj/machinery/singularity/attack_tk(mob/user)
+	to_chat(user, "<span class = 'notice'>You attempt to comprehend \the [src]...</span>")
+	spawn(rand(50,110))
```

Comment:
why is this spawn

@PJB3005 did you guys port timer subsystem at all

Reply:
No we didn't
But why the fuck would we use it for this when `spawn()` does a perfectly good job?

Reply:
TG has to dead horse beat every minor performance gain instead of being sane.

Reply:
I have a hard time believing that any timer system implemented inside BYOND will have better performance than an internal function

Reply:
Eh this is BYOND we're talking about to be fair.

Reply:
Isn't spawn one of those proven to be horrible ones though